jit-transfer-obj/src/main/java/org/dcsa/jit/transferobjects/PartyTO.java

```diff
@@ -7,7 +7,7 @@
 import java.util.UUID;
 
 public record PartyTO(
-  UUID id,
+  @Size(max = 100) String id,
```

Comment:
I don't think we expose the Party ID in the JSON?

Reply:
Not according to the swagger, no.

Reply:
ok, then I think this should be removed from the `PartyTO`

Reply:
Sounds good to me. Removed!